pages/login/index.js

```diff
@@ -82,12 +125,12 @@ border border-[#50545A] px-4 py-[1.1rem]"
             />
             <button
               onClick={login}
-              disable={loading}
+              disabled={signingIn}
               className={`relative mx-[1.1rem] h-[5rem] w-full rounded-[1rem] bg-gradient-to-b from-teal-400 to-teal-300 text-[1.8rem] text-black ${
-                loading ? 'cursor-wait opacity-50' : ''
+                signingIn ? 'cursor-wait opacity-50' : ''
```

Comment:
Use `mergeClassnames` helper function for dynamic class names since this causes weird issues sometimes @micohalvarez